tests/test_basic.py

```diff
@@ -4522,3 +4523,29 @@ def test_spontaneous_firing():
     numpy.testing.assert_allclose(ap_fall_time, ap_fall_time_expected)
     numpy.testing.assert_allclose(ap_rise_rate, ap_rise_rate_expected)
     numpy.testing.assert_allclose(ap_fall_rate, ap_fall_rate_expected)
+
+
+def test_register_feature():
+    """basic: Test register feature"""
+    import efel
+    efel.reset()
+
+    trace = {}
+    trace['T'] = numpy.arange(0, 100, 0.1)
+    trace['V'] = numpy.ones(len(trace['T'])) * -80.0
+    trace['stim_start'] = [25]
+    trace['stim_end'] = [75]
+
+    def test_feature():
```

Comment:
Could you rename this to not to start with test? Since pytest detects the functions that start with test it may be confusing. E.g. `a_feature` or `sample_feature` or something similar.